legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/legend-engine-xt-relationalStore-sqlDialectTranslation-pure/src/main/resources/core_external_store_relational_sql_dialect_translation/postgres/postgresSqlDialect.pure

```diff
@@ -185,41 +185,152 @@ function <<access.private>> meta::external::store::relational::sqlDialectTransla
 function <<access.private>> meta::external::store::relational::sqlDialectTranslation::postgres::postgresKeywords(): String[*]
 {
   [
-    // TODO: Update
-    'date'
-  ]
+    'all', 

```

Comment:
Why uppercased keywords in DuckDb and lowercased ones in Postgres? There should be some uniformity